editor-components/modal/confirmation-modal-1/confirmation.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from "react";
+import ComposerModalClose from "../../../composer-base-components/close/close";
+import { BaseModal, TypeUsableComponentProps } from "../../EditorComponent";
```

Comment:
remove unused import
```suggestion
import { BaseModal } from "../../EditorComponent";
```